scripts/status_reporter/aws_client.py

```diff
@@ -0,0 +1,30 @@
+import uuid
+
+import boto3
+
+
+class AwsClient:
+    def __init__(self, role_arn="arn:aws:iam::975596993436:role/storage-developer"):
+        self.role_arn = role_arn
+
+    def _assumed_role_session(self):
+        session = boto3.session.Session()
+        sts_client = session.client("sts")
+
+        assumed_role_object = sts_client.assume_role(
+            RoleArn=self.role_arn, RoleSessionName=str(uuid.uuid1().hex)
```

Comment:
If we wanted to be more informative, the getpass module has a getuser() method that returns the name of the currently logged in user, so sessions could be tagged with our macOS usernames.

But I don't think I've ever looked at these session names, so not a big deal.